src/httprpc.h

```diff
@@ -19,6 +23,9 @@ void InterruptHTTPRPC();
  */
 void StopHTTPRPC();
 
+/** Execute a single HTTP request, containing one or more JSONRPC requests. */
```

Comment:
ea98a42640b9ff77a462e55887025ddd1da54727

Might want to indicate `status` is an "out" param?

I notice this also bypasses our only authorization barrier for RPC which is an HTTP header, maybe you deal with this in another commit but that should probably be documented in the `ipcbind` help text